src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
@mtkocak I support to create a pure semantic generator. As I pointed out in another ticket in the long run I'd either want to see a bake result that does good practice without exception or one that does as little as possible, e.g. maybe just do html5 + normalize.css and be done with it.